### PR TITLE
OSDOCS-1854: Updating control plane component list

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1854

Preview: https://deploy-preview-34152--osdocs.netlify.app/openshift-enterprise/latest/security/tls-security-profiles?utm_source=github&utm_campaign=bot_dp

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kasturinarra / @wangke19 Can you please review this? We updated to also list the OpenShift control plane components that are included.

Preview: https://deploy-preview-34152--osdocs.netlify.app/openshift-enterprise/latest/security/tls-security-profiles?utm_source=github&utm_campaign=bot_dp

**Comment by ahardin-rh** [comment]
Looks great!

File: modules/tls-profiles-understanding.adoc [hunk 1]
```diff
@@ -26,7 +26,7 @@ For the Ingress Controller, the minimum TLS version is converted from 1.0 to 1.1
 ====
 
 |`Intermediate`
-|This profile is the recommended configuration for the majority of clients. It is the  default TLS security profile for the Ingress Controller, kubelet, and Kubernetes control plane. The profile is based on the link:https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29[Intermediate compatibility] recommended configuration.
+|This profile is the recommended configuration for the majority of clients. It is the  default TLS security profile for the Ingress Controller, kubelet, and control plane. The profile is based on the link:https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29[Intermediate compatibility] recommended configuration.
```
Comment (mburke5678) [inline modules/tls-profiles-understanding.adoc:29]: s/[id="tls-profiles-ingress-understanding_{context}"]/[id="tls-profiles-understanding_{context}"] to match the file name?

S/You can specify one of the following TLS security profiles/You can specify one of the following TLS security profiles for each component (to avoind folks thinking they need to set the same one for each??

File: security/tls-security-profiles.adoc [hunk 2]
```diff
@@ -10,7 +10,12 @@ TLS security profiles provide a way for servers to regulate which ciphers a conn
 Cluster administrators can choose which TLS security profile to use for each of the following components:
 
 * the Ingress controller
-* the Kubernetes control plane (Kubernetes API server, Kubernetes controller manager, and Kubernetes scheduler)
+* the control plane
```
Comment (mburke5678) [inline security/tls-security-profiles.adoc:13]: I *think* Ingress Controller takes an upper-case C.

Comment (bergerhoffer) [reply security/tls-security-profiles.adoc:13]: Good call, this was the only place I had it lowered.

**Pull request merged**